### PR TITLE
SIMD-0183: Skip Rent Rewrites

### PR DESCRIPTION
**Pull request by brooksprumo**
[Rendered view](https://github.com/solana-foundation/solana-improvement-documents/blob/69527f875b048e164f1d14b0f27bda541290207e/proposals/0183-skip-rent-rewrites.md)

### CODE REVIEW

File: proposals/0183-skip-rent-rewrites.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+If rent collection has *not* changed an account, the account must not be
+stored/written back in that slot.
+
+An important note, rent collection updates an account's `rent epoch` in addition
+to its balance.  If rent collection does not collect rent (i.e. the account's
+balance is unchanged), but *does* change the account's `rent epoch`, the
+account must still be stored.
+
+## Impact
+
```
Comment (HaoranYi) [inline proposals/0183-skip-rent-rewrites.md:52]: maybe, mention that account's delta hash and incremental snapshot hash will change, but full hash won't change?

**Review by HaoranYi — APPROVED** [review]
lgtm

**Review by jeffwashington — APPROVED** [review]
lgtm

**Comment by jumpsiegel** [comment]
lgtm..

**Review by jumpsiegel — APPROVED** [review]

**Pull request merged**